modules/main-rules/pages/terrain-effects-on-movement.adoc

```diff
@@ -18,8 +18,12 @@ include::partial$terrain-effects-table.adoc[]
 [[dangerous-terrain-test]]
 == Dangerous terrain test
 
-When you wish to move a unit into dangerous terrain, or when you wish to move a unit that is already in dangerous terrain, roll a D6: On a roll of 1, halt the unit at the edge of the terrain.
-If the unit is halted in this way, roll the D6 again: If you roll another 1 then the unit takes a hit.
+When you wish to move a unit into dangerous terrain, or when you wish to move a unit that is already in dangerous terrain, roll a D6:
+
+* On a result of 1: You may not move the unit any farther — leave it at the edge of the dangerous terrain.
+Or, where the unit is already in dangerous terrain, you may not move it.
+Then roll a D6 again: If you roll another 1 then the unit takes a hit.
+* On any other result: You may move the unit as normal.
```

Comment:
I tweaked this to use the 'may not move' wording to correlate with the changes to the Retreat move wording in `broken-detachments.adoc`. (Gets complicated, don't it, amazing how a small issue can snowball...!)

 It is also a bit clearer about 'moving into - stop' versus 'already in - don't start'.